.travis/install.sh

```diff
@@ -6,28 +8,33 @@ then
   then
     PIP_INSTALL_FLAGS="--user"
   fi
+else
+  SCAPY_SUDO="$SCAPY_SUDO -H"
 fi
 
-if python --version 2>&1 | grep -q PyPy; then
+$SCAPY_SUDO $PIP install $PIP_INSTALL_FLAGS -U mock
+
+if ! python --version 2>&1 | grep -q PyPy; then
   # cryptography requires PyPy >= 2.6, Travis CI uses 2.5.0
-  $SCAPY_SUDO pip install $PIP_INSTALL_FLAGS -U mock
-else
-  $SCAPY_SUDO pip install $PIP_INSTALL_FLAGS -U cryptography mock
+  $SCAPY_SUDO $PIP install $PIP_INSTALL_FLAGS -U cryptography
 fi
 
 # Install coverage
 if [ "$SCAPY_COVERAGE" = "yes" ]
 then
-  $SCAPY_SUDO pip install $PIP_INSTALL_FLAGS coverage
-  $SCAPY_SUDO apt-get install python-pyx
+  $SCAPY_SUDO $PIP install $PIP_INSTALL_FLAGS -U coverage
+  $SCAPY_SUDO $PIP install $PIP_INSTALL_FLAGS -U PyX
+  $SCAPY_SUDO $PIP install $PIP_INSTALL_FLAGS -U codecov
 fi
 
 # Install pcap & dnet
 if [ ! -z $SCAPY_USE_PCAPDNET ]
 then
   if [ "$TRAVIS_OS_NAME" = "linux" ]
   then
-    $SCAPY_SUDO apt-get install python-libpcap python-dumbnet openssl
```

Comment:
Do you have a good reason to remove `apt` based installation ? Somehow, this helped me catch a bug or two.

Note: if have no strong opposition to this change.

Reply:
We might want to switch to pip based packages for OS X too.

Reply:
The thing is when you use Python from Travis, you don't want to use the one installed with the system (it is always a 2.7, while we want to test with other versions too). When you use `apt-get`, you install packages for the Python installed with the system.